apps/meteor/client/views/root/hooks/useWordPressOAuth.ts

```diff
@@ -0,0 +1,91 @@
+import type { OauthConfig } from '@rocket.chat/core-typings';
+import { useSetting } from '@rocket.chat/ui-contexts';
+import { useEffect } from 'react';
+
+import { CustomOAuth } from '../../../../app/custom-oauth/client/CustomOAuth';
+
+const config: OauthConfig = {
```

Comment:
its not a good idea have a hook that mutates external variables, imagine if you use in multiple places. it could lead into a "race condition" or multiple hooks deleting and adding props